test/SourceOperation.spec.js

```diff
@@ -0,0 +1,37 @@
+/* global afterEach, before*/
+
+import { assert } from "chai";
+import fetchMock from "fetch-mock";
+
+import { setAuthenticationPromise } from "../src/api";
+import SourceOperation from "../src/model/SourceOperation";
+
+describe("Team", () => {
+  before(function() {
+    setAuthenticationPromise(Promise.resolve("testToken"));
+  });
+
+  afterEach(function() {
+    fetchMock.restore();
+  });
+
+  it("Get SourceOpewation", async () => {
```

Comment:
Cute, but we probably don't want baby talk in our code 🤔

Reply:
resolved with  8fff05b 😬